File: benchmarks/c/print_concat.c
#include <stdio.h>
#include <stdlib.h>

int main(){
    const char* env = getenv("PCOUNT");
    long count = env ? atol(env) : 200000;
    for (long i = 0; i < count; i++) {
        puts("hello");
        puts("world");
        puts("aether");
    }
    return 0;
}

File: benchmarks/rust/print_concat.rs
use std::env;

fn main() {
    let count: i64 = env::var("PCOUNT").ok().and_then(|v| v.parse().ok()).unwrap_or(200_000);
    let mut i: i64 = 0;
    while i < count {
        println!("hello");
        println!("world");
        println!("aether");
        i += 1;
    }
}
